newIDE/app/src/MainFrame/index.js

```diff
@@ -2564,6 +2563,8 @@ const MainFrame = (props: Props) => {
       // We were able to load and then save the project. We can now close the dialog,
       // open the project editors and check if leaderboards must be replaced.
       setNewProjectSetupDialogOpen(false);
+      setSelectedExampleShortHeader(null);
+      await setState(state => ({ ...state, createDialogOpen: false }));
```

Comment:
the logic you've done is in complete different order than before!
It's hard to get my head around the logic, but it seems fine this way...
I assume you've tested all cases to be safe?

Reply:
Intuitively looks better, resetting/closing at the end and the rest looks like a re-ordering 👍